src/Autofac.Extras.MvvmCross/AutofacMvxIocProvider.cs

```diff
@@ -387,7 +403,8 @@ public virtual void RegisterSingleton(Type tInterface, object theObject)
 
             // You can't inject properties on a pre-constructed instance.
             cb.RegisterInstance(theObject).As(tInterface).AsSelf().SingleInstance();
-            cb.Update(this.Container);
+
+            cb.Update(Container);
```

Comment:
I assumed that would be the case. I couldn't see anything of that sort with a quick look.